config/v3_4/translate/translate.go

```diff
@@ -28,31 +27,6 @@ func translateIgnition(old old_types.Ignition) (ret types.Ignition) {
 	return
 }
 
-func translateFileEmbedded1(old old_types.FileEmbedded1) (ret types.FileEmbedded1) {
```

Comment:
Hum, don't we need to still do that in 3.4.0?

Reply:
I dont think so, its not needed as there were no other overides outside of the 'mode' one. So there is no reason to expand the map.